drop-in/src/main/java/com/adyen/checkout/dropin/ui/paymentmethods/PaymentMethodsListViewModel.kt

```diff
@@ -84,8 +84,7 @@ class PaymentMethodsListViewModel(
                 type == null -> {
                     throw CheckoutException("PaymentMethod type is null")
                 }
-                PaymentMethodTypes.SUPPORTED_PAYMENT_METHODS.contains(type)
-                    && !PaymentMethodTypes.SUPPORTED_ACTION_ONLY_PAYMENT_METHODS.contains(type) -> {
```

Comment:
This means you have to check these (PIX) in the `getComponentAvailabilityCheckForType`.

Reply:
Yep, and `getComponentAvailabilityCheckForType` returns `null` for PIX which means it's always available.